xmtp_mls/migrations/2024-08-07-213816_create-private-preference-store/up.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE "private_preferences"(
+    -- The inbox_id the private preferences are owned by
+    "inbox_id" text NOT NULL,
+    -- Enum of the PRIVATE_PREFERENCE_TYPE (GROUP, INBOX, etc..)
+    "valueType" int NOT NULL,
```

Comment:
I think we will want to have some logic in the code that says if type is ADDRESS find the inbox_id for that address and actually only store and fetch inbox ids. Probably a migration needed to convert all the V2 items to inbox ids.